.gitignore

```diff
@@ -25,6 +25,11 @@ v8-compile-cache-0/
 dist/
 prebuilds/
 samples/**/pnpm-lock.yaml
+packages/zowe-explorer/**/.wdio-vscode-service
+packages/zowe-explorer/__tests__/**/.zowe
+packages/zowe-explorer/__tests__/**/.env
+packages/zowe-explorer/__tests__/__integration__/ci
+!packages/zowe-explorer/__tests__/__integration__/ci/zowe.config.json
```

Comment:
I've added the whole folder so that any new files are ignored if creating during testing. I felt that it was appropriate to only recognize changes to the config file itself (based on outcome of a conversation w/ @zFernand0), but I can add the schema if we want to factor in changes to that as well